### PR TITLE
304 highlight selected building on the map

### LINKED ISSUE #304: Highlight selected building on the map

### PR DESCRIPTION
**Pull request by longshuicy**
Update changed pin to svg and use theme color
image

---

For some reason using return coordinate works for me? 
I based off your original logic and switch the style to be a map pin. 
Comparison map also works.
Pasted Graphic 48

### CODE REVIEW

**Comment by Rashmil-1999** [comment]
One, teeeny bit of issue. I can't get it to go away

File: src/components/childComponents/map/Map.jsx [hunk 1]
```diff
@@ -247,24 +252,24 @@ const Map = (props) => {
 						.then((response) => response.json())
```
Comment (Rashmil-1999) [inline src/components/childComponents/map/Map.jsx:252]: Also, the url is generated regardless so we might have to just look at the response to decide whether to clear or not.

**Comment by longshuicy** [comment]
> One, teeeny bit of issue. I can't get it to go away

hahah oops i forgot about that! Will address that.

File: src/components/childComponents/map/Map.jsx (hunk 1 above)
Comment (longshuicy) [reply src/components/childComponents/map/Map.jsx:252]: addressed

**Pull request merged**